tox.ini

```
@@ -3,19 +3,12 @@ skipsdist = True
 usedevelop = True
 
 envlist =
-    python{3.8,3.9}-django{3.2}-wagtail{4.1,4.2}-{sqlite,postgres}
-    python{3.10}-django{3.2}-wagtail{4.1,4.2,main}-{sqlite,postgres}
-    python{3.8,3.9,3.10}-django{4.0}-wagtail{4.1,4.2,main}-{sqlite,postgres}
-    python{3.8,3.9,3.10,3.11}-django{4.1,main}-wagtail{4.1,4.2,main}-{sqlite,postgres}
-    python{3.8,3.9,3.10,3.11}-django{4.2,main}-wagtail{5.0,5.1,main}-{sqlite,postgres}
+    python{3.11,3.12}-django{3.2,4.0,4.1,4.2,main}-wagtail{5.2,main}-{sqlite,postgres}
```

```suggestion
    python{3.11,3.12}-django{4.2,main}-wagtail{5.2,main}-{sqlite,postgres}
```
at a minimum. 
I would suggest splitting so that Django main is only tested with Wagtail main